packages/UI/src/TextInput.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { InputHTMLAttributes, forwardRef } from "react";
```

Comment:
import react är oanväd och Input HTMLAttrubutes måste importeras som en typ
```suggestion
import { forwardRef } from "react";
import type { InputHTMLAttributes } from "react";
```